samples/file-encryptor/enclave/mbedtls_src/keys.cpp

```diff
@@ -12,8 +12,9 @@
 #include <mbedtls/rsa.h>
 #include <mbedtls/sha256.h>
 
-#include "common.h"
-#include "encryptor.h"
+#include "aes_context.h" // Must be included before common/encryptor.h
```

Comment:
Move this include inside encryptor.h

Reply:
Same as the suggestion above.

Reply:
removed header aes_context.h and added struct definition in encryption.cpp